aten/src/ATen/core/function_schema_inl.h

```diff
@@ -51,6 +51,35 @@ inline std::ostream& operator<<(std::ostream& out, const FunctionSchema& schema)
   return out;
 }
 
+inline bool Argument::isBackwardCompatibleWith(
+      const Argument& old,
+      bool is_return,
+      std::ostream* why_not) const {
+    const Argument* lhs = this;
+    const Argument* rhs = &old;
+    // functions are covariant in arguments but contravariant in returns
+    if (is_return) {
```

Comment:
I looked and _every call site has this as a constant_, so this can simply be accomplished by just swapping the order of the call for the returns. In face this comment makes much more sense inside the return handling then here.

Reply:
Sure, I was debating with myself, whether I should keep the is_return in this function. No strong preference. Updated.